test/e2e/step-definitions/dc.steps.js

```diff
@@ -575,3 +575,29 @@ Then(/^I scroll to the "([^"]*)" header$/, async function (header) {
     element.scrollIntoView();
   }, xpath);
 });
+
+Then(/^I close Onetrust pop up if present$/, async function () {
+  const btnEnableAll = await this.page.native.locator('#onetrust-accept-btn-handler');
+  await expect(btnEnableAll).toBeVisible({timeout: 30000});
+  btnEnableAll.click();
```

Comment:
This errors out if the OneTrust banner does not appear in the page (e.g, if the test runs in US). Please add a condition.